ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -141,8 +149,12 @@ public static String getInstanceName(String host, Integer port) {
     JSONArray all = root.getJSONArray(ZKINFO_STR);
     for (int i = 0; i < all.length(); i++) {
       JSONObject entry = all.getJSONObject(i);
+      String name = entry.getString(DATACENTER_STR);
       byte id = (byte) entry.getInt(DATACENTER_ID_STR);
-      DcZkInfo dcZkInfo = new DcZkInfo(entry.getString(DATACENTER_STR), id, entry.getString(ZKCONNECTSTR_STR));
+      ReplicaType replicaType = entry.optEnum(ReplicaType.class, REPLICA_TYPE_STR, ReplicaType.DISK_BACKED);
```

Comment:
That would break backwards compatibility with existing configs by adding a new required field, replicaType. Personally, I don't find having default values too complicated. I also wanted to resist having a dummy value for cloud_backed datacenters, since the work I'm doing here is admittedly a little transient. Eventually, the zk string for cloud dcs will likely be set to the VCR zk for things like getting partition override info and additional metadata.